### DISCUSSION TITLE
Breakout into Districts

### DISCUSSION THREAD

**Discussion by wendydunning** [code4recovery/12-step-meeting-list#1158] [Q&A]
Hi, I've been using the plugin for years. Beyond awesome. I love the new UI. I was just wondering if there is a way to separate meetings out into Districts without having to put it on a "custom link" [search]. Is there a search term I could use to turn the Districts into distinct pages? It's for an al-anon intergroup [service center] that serves 9 districts. Sorry if I am not explaining very well. I know enough code to be dangerous and don't want to screw anything up!
Thanks in advance,

Wendy

**Comment by joshreisner** [comment]
Hi Wendy, sounds like you're asking about TSML UI not supporting the "switch to districts" option in the Anywhere dropdown, and how you have to create your own menu of districts to link people to district-level lists? 

If so, we don't currently have a plan to implement that feature in TSML UI, but believe that it should be able to work around this limitation pretty easily in your own site navigation - if you could share a link to your website we can better advise

**Comment by wendydunning** [comment]
Wow, thanks for getting back to me on a Sunday!

Link to the site is https://alanonva.com/meetings

I've made an alternate menu bar for the top that executes the searches
okay, based on hints I picked up in the discussions.

The problem is that it does not create an actual 'page' that I can add
other content to [such as a calendar for each district's events]--
It could be that I'm just over my head or this is completely a 'wordpress'
problem and not a TSML issue. If that's the case feel free to say so, but
I'd appreciate any pointing in any direction!!

Thanks and have a good one,

Wendy

**Reply by joshreisner** [reply]
oh ok! glad i got this extra detail - sounds like a different issue then. my advice would be to go ahead and create a prototype district landing page in wordpress, and get the other content on there, and we can probably figure out a way to use the `[tsml_ui]` shortcode to embed the district-level list

**Comment by wendydunning** [comment]
Excellent. Thanks so much. I already have some of the district pages
started.
Looking forward to shortcodes!

Thanks again - no rush, but it will be awesome!
